native/go-1.4/Makefile

```diff
@@ -0,0 +1,26 @@
+PKG_NAME = go
+PKG_VERS = 1.4.2
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)$(PKG_VERS).src.$(PKG_EXT)
+PKG_DIST_SITE = https://storage.googleapis.com/golang
+PKG_DIR = $(PKG_NAME)
+
+DEPENDS =
+
+HOMEPAGE   = http://golang.org/
+COMMENT    = Go is an open source programming language that makes it easy to build simple, reliable, and efficient software.
+LICENSE    =
+
+CONFIGURE_TARGET = nop
+COMPILE_TARGET = nop
+INSTALL_TARGET = myComp
+
+include ../../mk/spksrc.native-cc.mk
+
+myComp:
+	@$(MSG) "Building Go"
+	cd $(WORK_DIR)/$(PKG_NAME)/src && ./make.bash
+	@$(MSG) "Building Go for arm"
```

Comment:
Seeing as this package is purely for bootstrapping 1.6, L21/L22 would suffice, right? No need to build ARM or amd64: amd64 is what's built when running `./make.bash`, and that's all you'd need to bootstrap the other native package.